champions/src/main/java/me/mykindos/betterpvp/champions/listeners/ArrowListener.java

```diff
@@ -57,6 +58,16 @@ public void onBaseArrowDamage(CustomDamageEvent event) {
         }
     }
 
+    @EventHandler(priority = EventPriority.LOWEST)
+    public void onProjectileDelay(PreCustomDamageEvent event) {
+        if (event.getCustomDamageEvent().getProjectile() == null) {
+            return;
+        }
+
+        event.getCustomDamageEvent().setForceDamageDelay(0);
+        event.getCustomDamageEvent().setDamageDelay(0);
+    }
+
```

Comment:
Remove the projectile check here: https://github.com/Mykindos/BetterPvP/blob/c540414892f40ef8cb725789915e50b161081efb/core/src/main/java/me/mykindos/betterpvp/core/combat/listeners/CombatListener.java#L375